graphsdk/src/androidTest/java/com/microsoft/graph/functional/OutlookTests.java

```diff
@@ -41,4 +38,22 @@ public void testSendMail() {
         message.toRecipients = recipients;
         testBase.graphClient.getMe().getSendMail(message, true).buildRequest().post();
     }
+
+    public void testGetFindMeetingTimes() {
+        TestBase testBase = new TestBase();
+        List<AttendeeBase> attendees = new ArrayList<>();
+        AttendeeBase attendeeBase = new AttendeeBase();
+        EmailAddress email = new EmailAddress();
+        email.address = "katiej@mod810997.onmicrosoft.com";
```

Comment:
I am currently using a constants file and I should use this for Planner and OneNote tests. In this instance, I should probably be getting this address programmatically so it works on any tenant instead of hard coding an email.